### PR TITLE
😫

### PR DESCRIPTION
**Pull request by adrianschmidt**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Review by jgroth — APPROVED** [review]

File: src/components/picker/examples/picker-empty-suggestions.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import { LimelPickerCustomEvent, ListItem } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
 
+// 👇 Do NOT copy this to production code! It's a MOCK for this example ONLY!
```
Comment (Kiarokh) [inline src/components/picker/examples/picker-empty-suggestions.tsx:4]:
```suggestion
// 👇 Do NOT copy and paste this to production code! It's a MOCK for this example ONLY!
```

Comment (Kiarokh) [reply src/components/picker/examples/picker-empty-suggestions.tsx:4]: I think you should write it so, to be more "correct".

**Comment by Kiarokh** [comment]
😄

**Review by anderssonjohan — APPROVED** [review]

**Review by vicgeralds — COMMENTED** [review]
Maybe, since this turns out to be a problem (I assume), we should instead/also do some changes to the code to make it clearer that we simulate a fake request.

What about

```ts
function fakeNetworkRequest(fakeResponseData = []) {
    return new Promise((resolve) => 
        setTimeout(() => resolve(fakeResponseData), NETWORK_DELAY)
    )
}
```

It would then be easier to spot `await fakeNetworkRequest()` and understand you shouldn't copy that line,

compared to seeing

```ts
// Simulate some network delay
setTimeout(() => {
   ...
}, NETWORK_DELAY);
```

everywhere and not really understanding what it does.

We could perhaps even move `fakeNetworkRequest` to another module, so they can't easily copy-paste it and they will be forced to remove the call to it.

**Comment by jgroth** [comment]
> Maybe, since this turns out to be a problem (I assume), we should instead/also do some changes to the code to make it clearer that we simulate a fake request.
> 
> What about
> 
> ```ts
> function fakeNetworkRequest(fakeResponseData = []) {
>     return new Promise((resolve) => 
>         setTimeout(() => resolve(fakeResponseData), NETWORK_DELAY)
>     )
> }
> ```
> 
> It would then be easier to spot `await fakeNetworkRequest()` and understand you shouldn't copy that line,
> 
> compared to seeing
> 
> ```ts
> // Simulate some network delay
> setTimeout(() => {
>    ...
> }, NETWORK_DELAY);
> ```
> 
> everywhere and not really understanding what it does.
> 
> We could perhaps even move `fakeNetworkRequest` to another module, so they can't easily copy-paste it and they will be forced to remove the call to it.

Haha, this is more less exactly what I said to Adrian over Slack :smile:

File: src/components/picker/examples/picker-empty-suggestions.tsx (hunk 1 above)
Comment (vicgeralds) [inline src/components/picker/examples/picker-empty-suggestions.tsx:4]: I don't know, I'm a bit concerned that all these warning don't have a nice tone and assume the reader is stupid. We should be very clear and make it difficult to make this mistake (copying what should definitely not be copied), but in a friendly way if possible and without distracting too much from the example code.

Comment (adrianschmidt) [reply src/components/picker/examples/picker-empty-suggestions.tsx:4]: True…

I just didn't feel I had the time to do the better solution right away, but I agree. It's perhaps better to not merge this in the interim, and just do the better solution when someone has a little bit of time 👍

**Pull request closed without merging**